FastExcel/FastExcel.cs

```diff
@@ -109,9 +109,9 @@ internal void CheckFiles()
                 }
                 else if (!ExcelFile.Exists)
                 {
-                    string exceptionMessage = string.Format("Input file '{0}' does not exist", ExcelFile.FullName);
+                    var exceptionMessage = $"Input file '{ExcelFile.FullName}' does not exist";
                     ExcelFile = null;
-                    throw new Exception(exceptionMessage);
+                    throw new FileNotFoundException(exceptionMessage);
```

Comment:
@mrjono1, hello!

Changed the type of exception. This is normal?

Reply:
looks like a good fix